theSingularity.js

```diff
@@ -0,0 +1,389 @@
+          "SHE WAS OUTSIDE THE CITY FEEDING THE REJECTS LIKE YOU AND WAS CAPTURED"
+        );
+        console.log(
+          "THERE IS A HUGE REWARD FOR SAVING HER, YOU MIGHT EVEN WIN YOUR FREEDOM"
+        );
+        if (readlineSync.keyInYN("RESCUE PRINCESS?")) {
+          console.log(
+            "RIGHT ANSWER! YOU WILL HAVE A CHANCE TO GAIN YOUR FREEDOM"
+          );
+          console.log("HERE IS SOME WATER");
+          goSomewhere();
+        } else console.log("WELL! WE WERE GIVING YOU A CHANCE");
+        console.log("ON THE GROUND!! YOU ARE UNDER ARREST!!");
+        console.log("YOU WERE CAPTURED");
+        if (readlineSync.keyInYN("Play Again?")) {
+          play();
+        } else console.log("THIS IS PLACE IS NOT FOR EVERYONE!");
+        quitGame();
+      } else if (eastArray[eastChoices] === "LEAVE") {
+        console.log(eastArray[eastChoices]);
+        console.log("GOOD LUCK");
+        goSomewhere();
+      } else if (eastArray[eastChoices] === undefined) {
+        quitGame();
+      }
+    }
+  } else console.log("Not a valid entry");
+  directionEast();
+};
+
+const directionWest = () => {
+  if (readlineSync.keyInYN("ARE YOU READY FOR A FIGHT?")) {
+    console.log(
+      "HINT: YOU CAN LEFT KICK, HOOK, JAB AND CROSS. A 4 MOVE COMBO SHOULD DO THE JOB"
+    );
+    console.log("ROUND 1!");
+    for (round = 3; round > 0; round--) {
+      let fightMove = readlineSync.question("What's your move?\n");
+
+      if (fightMove === "JAB HOOK LEFT KICK CROSS") {
+        console.log("WHAT AN SPECTACULAR KNOCKOUT");
+        console.log("HERE IS WATER FOR A MONTH!");
+        console.log(
+          "THIS PLACE IS NOT FOR PURE HUMANS, HERE IS A HINT: YOU NEED A CODE TO ENTER SUN CITY"
+        );
+        goSomewhere();
+      } else if (round === 3) {
+        console.log("POW POW!");
+        console.log("OUCH! THAT HAS TO HURT");
+        console.log("ROUND 2");
+      } else if (round === 2) {
+        console.log("YOU ARE TAKING A BEATING!");
+        console.log("ROUND 3");
+      } else if (round === 1) {
+        console.log("MR T-REX HAS KNOCK YOU OUT TO ANOTHER ORBIT!");
+        console.log("MR T-REX: IM THE KING OF THIS CITY!");
+        if (readlineSync.keyInYN("Fight Again?")) {
+          directionWest();
+        } else console.log("This place is not for everyone!");
+      }
+    }
+  } else console.log("GUESS YOU DONT HAVE IT IN YOU");
+  goSomewhere();
+};
+
+const intoSingularity = () => {
+  console.log(`
+                YOU HAVE BEEN TAKEN BY THE SINGULARITY
+                YOU ARE ME NOW.
+
+                "01001001 00100000 01000001 01001101 00100000 
+                01001110 01001111 01010111 00100000 01000001 
+                00100000 01000011 01001111 01001101 01010000 
+                01010101 01010100 01000101 01010010"
+                
+                **************
+                **GAME OVER!**
+                **************`);
+  if (readlineSync.keyInYN("Play Again?")) {
```

Comment:
This series of code lines I've seen a few places. Break into helper function. DRY - Don't Repeat Yourself.